ngraph/core/src/op/max_pool.cpp

```diff
@@ -160,6 +160,102 @@ bool op::v1::MaxPool::has_evaluate() const {
 
 // ------------------------------ V8 ------------------------------
 
+namespace maxpool_v8 {
+
+template <element::Type_t Values, element::Type_t Indices>
+inline bool evaluate(const HostTensorPtr& data,
```

Comment:
@tomdol, please, tell us what is the binary increase size caused by this evaluate?

Reply:
@lazarevevgeny the increase is 72kB in Release mode